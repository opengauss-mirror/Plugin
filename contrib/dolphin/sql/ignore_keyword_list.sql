create schema ignore_keyword_list;
set current_schema to 'ignore_keyword_list';

create schema ignore_keyword_list;
set current_schema to 'ignore_keyword_list';
select 1 interval; --error
set disable_keyword_options = 'interval';
select 1 interval; --ok
set disable_keyword_options = 'intervalxx';
set disable_keyword_options = 'interval,interval';
select 1 interval; --ok
reset disable_keyword_options;
select 1 interval; --error
drop schema ignore_keyword_list cascade;

drop schema ignore_keyword_list cascade;
reset current_schema;
