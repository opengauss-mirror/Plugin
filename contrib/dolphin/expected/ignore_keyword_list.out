create schema ignore_keyword_list;
set current_schema to 'ignore_keyword_list';
select 1 interval; --error
ERROR:  syntax error at or near "interval"
LINE 1: select 1 interval;
                 ^
set disable_keyword_options = 'interval';
select 1 interval; --ok
 interval 
----------
        1
(1 row)

set disable_keyword_options = 'intervalxx';
WARNING:  invalid disable keyword "intervalxx", will be skipped.
set disable_keyword_options = 'interval,interval';
select 1 interval; --ok
 interval 
----------
        1
(1 row)

reset disable_keyword_options;
select 1 interval; --error
ERROR:  syntax error at or near "interval"
LINE 1: select 1 interval;
                 ^
drop schema ignore_keyword_list cascade;
